frotz.py

```diff
@@ -126,15 +131,45 @@ def start_game(self, game_file):
             # print a welcome banner and pause for a moment
             # that way the end user will have a chance to see which version of frotz we are using
             # and which file we are loading
-
-            if platform.machine().startswith('arm'):
-                logging.error('ARM not supported, yet') # FIXME
-                sys.exit(0)
+            if platform.version().find("Ubuntu") > -1 or platform.version().find("Debian") > -1:
+                cmd = '/usr/games/frotz'
             else:
-                if platform.architecture()[0] == '64bit':
-                    self._vte.feed_child("cd '%s'; clear; frotz64|head -3 ; echo '\nLoading %s...'; sleep 2; frotz64 '%s'; exit\n" % (save_dir, os.path.basename(game_file), game_file))
-                else:
-                    self._vte.feed_child("cd '%s'; clear; frotz32|head -3 ; echo '\nLoading %s...'; sleep 2; frotz32 '%s'; exit\n" % (save_dir, os.path.basename(game_file), game_file))
+                cmd = '/usr/bin/frotz'
+            if os.path.isfile(cmd) and os.access(cmd, os.X_OK):
+                logging.debug('Using frotz installed in the system')
+                self._vte.feed_child(
+                    (
```

Comment:
Indentation here is very strange; the open parenthesis is not aligned with the close.

Reply:
Done in 480c207